### ISSUE TITLE
Schema validation for cmake-variants.yaml doesn't work

### ISSUE THREAD

**Issue by TSlivede** [microsoft/vscode-cmake-tools#2135]
### Brief Issue Summary

https://vector-of-bool.github.io/docs/vscode-cmake-tools/variants.html says:
>Note:      CMake Tools provides a YAML validation schema, which is only checked in the editor when using the YAML Support by Red Hat extension.

I installed [YAML Language Support](https://github.com/redhat-developer/vscode-yaml) but get the error:
```
Cannot Load content for: cmake-tools-schema:///schemas/variants-schema.json. Unknown schema: 'cmake-tools-schema'
```

### Expected:

1. Create file `cmake-variants.yaml` in project root directory 
2. open file
3. get schema based auto completion

### Apparent Behavior:

1. Create file `cmake-variants.yaml` in project root directory 
2. open file
3. click on `variants-schema.json` shown directly above line 1
3. get error listed above.

### CMake Tools Log

```
[variant] Loaded new set of variants
[kit] Successfully loaded 8 kits from /home/user/.local/share/CMakeTools/cmake-tools-kits.json
```

### Other Logs

The output window for "YAML Support" is completely empty.

### Platform and Versions

- **Operating System**: Ubuntu 20.04.3 LTS
- **CMake Version**: 3.16
- **VSCode Version**: 1.16.1
- **CMake Tools Extension Version**: 1.8.1
- **Compiler/Toolchain**: GCC 9.3
- Red Hat YAML Language Support: 0.22.0

### Other Notes/Information

`~/.vscode/extensions/ms-vscode.cmake-tools-1.8.1/dist/schema` contains schemas in several languages

A created `cmake-variants.json` file gets successfully validated against the schema.

**Comment by andreeis** [comment]
@TSlivede, thank you for opening this report. We reproduce the problem. We will investigate and fix this.

**Comment by gcampbell-msft** [comment]
@TSlivede This should get resolved by #3720
